Gordon360/Controllers/ProfilesController.cs

```diff
@@ -611,5 +629,18 @@ public ActionResult<IEnumerable<MembershipHistoryViewModel>> GetMembershipHistor
 
             return Ok(membershipHistories);
         }
+
+        /// <summary>
+        /// Return a list of mail destinations' descriptions.
+        /// </summary>
+        /// <returns> All Mail Destinations</returns>
+        [HttpGet]
+        [Route("mailstops")]
+        public ActionResult<IEnumerable<string>> GetMailStops()
+        {
+            var mail_stops = _webSQLContext.Mailstops.Select(m => m.code)
```

Comment:
I agree that it looks like it belongs in the Service instead of Controller.
I had based myself of some examples that return just a list of things instead of a view model and were written in the Controller.
One of the many examples I found was the route to get the Buildings options for the drop down in people search :
![image](https://github.com/gordon-cs/gordon-360-api/assets/115328139/c1414ef4-3b37-4d9d-adc6-3ec311a973dc)
In this cases, is it ok to be like it is in the controllers, should it have been made in the service?

Reply:
I think for now it's ok to leave it in the controller. The best practice is to put it in the service, both to achieve clean separation of concerns, and also to make the code easier to re-use within the API.